src/layouts/EditNavBar.jsx

```diff
@@ -612,25 +621,47 @@ const EditNavBar = ({ match }) => {
 
   return (
     <>
-      {showDeletedText && !isEmpty(deletedLinks) && (
+      {!isEmpty(deletedLinks) && (
         <GenericWarningModal
+          isOpen={isRemovedOpen}
+          onClose={onRemovedClose}
           displayTitle="Removed content"
           displayText={`Some of your content has been removed as they attempt to link to invalid folders. No changes are permanent unless you press Save on the next page.<br/>${deletedLinks}`}
-          onProceed={() => {
-            setShowDeletedText(false)
-          }}
-          proceedText="Acknowledge"
-        />
+        >
+          <Button onClick={onRemovedClose}>Acknowledge</Button>
+        </GenericWarningModal>
       )}
       {itemPendingForDelete.id && (
-        <DeleteWarningModal
-          onCancel={() => setItemPendingForDelete({ id: null, type: "" })}
-          onDelete={() => {
-            deleteHandler(itemPendingForDelete.id)
+        <GenericWarningModal
+          isOpen={isDeleteOpen}
+          onClose={() => {
             setItemPendingForDelete({ id: null, type: "" })
+            onDeleteClose()
           }}
-          type={itemPendingForDelete.type}
-        />
+          displayTitle={`Delete ${itemPendingForDelete.type}`}
+          displayText={`Are you sure you want to delete ${itemPendingForDelete.type}?`}
+        >
+          <Button
+            variant="ghost"
```

Comment:
same as before - intended variant might be `clear`

Reply:
https://github.com/isomerpages/isomercms-frontend/pull/941/commits/a4e792aaffbb26913a0cc9a40493c17faf42a5d7